src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -3,23 +3,64 @@ import environment from 'platform/utilities/environment';
 import { apiRequest } from 'platform/utilities/api';
 import content from '../locales/en/content.json';
 
-export const fetchFacilities = async ({
+const joinAddressParts = (...parts) => {
+  return parts.filter(part => part != null).join(', ');
+};
+
+const formatQueryParams = ({
   lat,
   long,
-  radius = 500,
```

Comment:
Removed these default values and the function will just treat them as optional. In the facility search component we are passing them when we need them.